examples/android/android.rs

```diff
@@ -1,9 +1,26 @@
-use bevy::prelude::*;
+use bevy::{
+    prelude::*,
+    render::{
+        render_resource::WgpuLimits,
+        settings::{WgpuSettings, WgpuSettingsPriority},
+    },
+};
 
 // the `bevy_main` proc_macro generates the required android boilerplate
 #[bevy_main]
 fn main() {
     App::new()
+        // This configures the app to use the most compatible rendering settings.
+        // They help with compatibilty with as many devices as possible.
+        .insert_resource(WgpuSettings {
+            priority: WgpuSettingsPriority::Compatibility,
+            limits: WgpuLimits {
+                // This allows for devices with bigger screens
+                max_texture_dimension_2d: 8192,
+                ..WgpuLimits::downlevel_defaults()
```

Comment:
Based on my understanding of the situation, this type of override should generally only be used during development. It forces you to "code to" highly compatible limits. When deploying an app though, we should let the device use exactly the limits that it supports. The default settings should already allow an "already written to be compatible" app to run in as many places as possible. And using the defaults (going up to the current device's limits) should allow maximal perf / quality for the given device.

Reply:
That was my understanding too, but it didn't work on my android tablet.

I think for an example in Bevy, it can make sense to maximise chances that it can run.

Reply:
If I remember correctly, it was failing on the `max_uniform_buffer_binding_size`. I don't have access to that tablet for now to check

Reply:
My bad, I retested and it works, I must have tested the limits before setting the compatibility priority. Works without setting them 👍